middlewares/validators/authValidator.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable newline-per-chained-call */
+// const emailExist = require('../../lib/utils/checkers/emailExistChecker');
+// const userModelManager = require('../../lib/modelManagers/usermodel');
+
+const errorMessage = (res, message) => res.status(400).json({
+  status: false,
+  message
+});
+
+/**
+   * @description check against errors on signup fields
+   * @param {object} req
+   * @param {object} res
+   * @param {object} next
+   * @returns {object} returns error object if error exist
+   */
+const authValidator = (req, res, next) => {
+  req.check('email')
```

Comment:
Abstracting this into a function make the code cleaner and more reusable.

Reply:
done